interface/src/project/SettingsCustomization.tsx

```diff
@@ -464,7 +471,7 @@ const SettingsCustomization: FC = () => {
                           setMasks(['']);
                         }}
                       >
-                        <ToggleButton value="8" disabled={(de.m & 1) !== 0 || de.id === ''}>
+                        <ToggleButton value="8" disabled={(de.m & 1) !== 0 || de.id === '' || de.s === de.id}>
```

Comment:
i think the check for `=== ''` is not needed anymore.

Reply:
yes, good spot